charts/karpenter/templates/clusterrole.yaml

```diff
@@ -22,12 +22,18 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["storageclasses", "csinodes"]
     verbs: ["get", "watch", "list"]
+  - apiGroups: [""]
+    resources: ["replicationcontrollers"]
```

Comment:
Nitpicking because these files are critically important to customers. Can we move this to line 20. This file is organized into objects that share the same API Group and Permissions, and replication controllers is equivalent to pods/nodes/persistentvolumes/persistentvolumeclaims.

List is a superset of get, so you could remove the `get` permission above, or explicitly add it to the `replicationcontroller` rule here.